### PR TITLE
Remove duplicate face

### PR DESCRIPTION
**Pull request by baskerville**

### CODE REVIEW

File: zenburn-theme.el [hunk 1]
```diff
@@ -581,7 +581,6 @@ Also bind `class' to ((class color) (min-colors 89))."
    `(ledger-font-other-face ((t (:foreground ,zenburn-fg))))
    `(ledger-font-posting-account-face ((t (:foreground ,zenburn-blue-1))))
    `(ledger-font-posting-account-cleared-face ((t (:foreground ,zenburn-fg))))
-   `(ledger-font-posting-account-pending-face ((t (:foreground ,zenburn-orange))))
    `(ledger-font-posting-amount-face ((t (:foreground ,zenburn-orange))))
    `(ledger-font-posting-account-pending-face ((t (:foreground ,zenburn-orange))))
```
Comment (bbatsov) [inline zenburn-theme.el]: To keep the alphabetic order I'd remove this one instead.

Comment (baskerville) [inline zenburn-theme.el]: Please go ahead.

**Comment by baskerville** [comment]
Should be fine now.

**Comment by bbatsov** [comment]
:+1: Thanks!

**Pull request merged**